Cargo.toml

```diff
@@ -32,7 +32,7 @@ amplify_derive = "~0.1.2"
 lazy_static = "~1.4.0"
 bech32 = "~0.7.2"
 ed25519-dalek = "~1.0.0-pre.3"
-bitcoin = { git = "https://github.com/LNP-BP/rust-bitcoin", branch = "staging" }
+bitcoin = { git = "https://github.com/LNP-BP/rust-bitcoin", branch = "staging", features = ["rand"] }
```

Comment:
We will need this only in test configuration, so can you pls move this feature to `dev-dependencies` it as described in https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

Reply:
Done